installers/metrics/ansible/roles/pgo-metrics/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Set Metrics Directory Fact
+  set_fact:
+    metrics_dir: "{{ ansible_env.HOME }}/.pgo/metrics/{{ metrics_namespace }}"
+  tags: always
+
+- name: Ensure Output Directory Exists
+  file:
+    path: "{{ metrics_dir }}"
+    state: directory
+    mode: 0700
+  tags: always
+
+- include_tasks: "{{ tasks }}"
+  loop:
+    - openshift_auth.yml
+    - openshift.yml
+  loop_control:
+    loop_var: tasks
+  when: openshift_host != ''
+  tags: always
+
+- include_tasks: "{{ tasks }}"
+  loop:
+    - kubernetes_auth.yml
+    - kubernetes.yml
+  loop_control:
+    loop_var: tasks
+  when: kubernetes_context != '' or kubernetes_in_cluster | bool
+  tags: always
+
+- name: Use kubectl or oc
+  set_fact:
+    kubectl_or_oc: "{{ openshift_oc_bin if openshift_oc_bin is defined else 'kubectl' }}"
+  tags: always
+
+- include_tasks: cleanup.yml
+  tags:
+    - update-metrics
+    - uninstall-metrics
+
+- include_tasks: kubernetes_cleanup.yml
+  when: kubernetes_context != '' or kubernetes_in_cluster | bool
+  tags:
+    - uninstall-metrics
+
+- include_tasks: openshift_cleanup.yml
+  when: openshift_host != ''
+  tags:
+    - uninstall-metrics
+
+- name: Install Crunchy PostgreSQL Operator Metrics
+  block:
+    - name: Download pgmonitor {{ pgmonitor_version }}
+      get_url:
+        url: https://github.com/CrunchyData/pgmonitor/archive/{{ pgmonitor_version }}.tar.gz
+        dest: "{{ metrics_dir }}"
+        mode: "0600"
+      tags:
```

Comment:
tags are defined for the block so I don't think this will be used

Reply:
These tags have been removed.